components/table/ReportTable.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Flex } from "@chakra-ui/layout"
+import { TimesheetEntry } from "@/lib/types/apiTypes"
+
+interface ReportTableProps {
+    entries: TimesheetEntry[]
+}
+
+function ReportTable({ entries }: ReportTableProps): JSX.Element {
+    const responseLength = entries.length
+    const totalQuantity = entries.reduce(
+        (total, currentItem) => (total += currentItem.quantity),
+        0
+    )
+
```

Comment:
I'd remove `entries[0].date` and `entries[responseLength - 1].date` from the JSX of the return statement below below and introduce the dates as constants here:
```
const startDate = entries[0].date
const  endDate = entries[responseLength - 1].date
```
Then use those in `return`.